### DISCUSSION TITLE
how to send a simple in-app notification ?

### DISCUSSION THREAD

**Discussion by ayoami** [norkunas/onesignal-php-api#176] [Q&A]
Hi, i have implemented the library correctly and i could get the my apps list, but i dont know how to send a simple in-app notification to all my users with a title, body, link and a image like onesignal dashboard. thanks

**Comment by norkunas** [comment]
Hello,

Basically by sending payload via notifications endpoint? :slightly_smiling_face: 
Each API POST/PUT etc. response expects some payload, so you look at the OneSignal official docs what it does accept and provide that.
Also there is a basic example in README https://github.com/norkunas/onesignal-php-api#notifications-api

**Reply by ayoami** [reply]
Sorry for the misunderstanding. I'm referring to sending an in-app notification, which is a popup within the app. Thanks

**Reply by norkunas** [reply]
In docs there aren't any related info about sending in-app notifications through api or I've missed something, but probably it's not possible.
But isn't it the same as just sending notification?

**Reply by ayoami** [reply]
Actually, the in-app option is better because it launches when the user is in the app, resulting in a higher click-through rate (CTR) compared to a simple notification

**Reply by norkunas** [reply]
I see. But as I said, I didn't find any related info in docs for now, so If you'll find how to do that through api, I'd check it and update library if needed
